viewer/packages/camera-manager/src/DefaultCameraManager.ts

```diff
@@ -0,0 +1,742 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import TWEEN from '@tweenjs/tween.js';
+import { ComboControls } from './ComboControls';
+import {
+  CameraManagerCallbackData,
+  CameraControlsOptions,
+  CameraChangeData,
+  PointerEventDelegate,
+  CameraState
+} from './types';
+import { CameraManager } from './CameraManager';
+import { assertNever, EventTrigger, InputHandler, disposeOfAllEventListeners } from '@reveal/utilities';
+import range from 'lodash/range';
+
+export class DefaultCameraManager implements CameraManager {
```

Comment:
Add description about how the default camera manage behaves and what it supports.